crates/weaver_forge/templates/test/event.md

```diff
@@ -1,10 +1,10 @@
-{%- set file_name = ctx.root_namespace | snake_case -%}
+{%- set file_name = ctx.event_namespace | snake_case -%}
 {{- template.set_file_name("event/" ~ file_name ~ ".md") -}}
 
-## Events Namespace `{{ ctx.root_namespace }}`
+## Events Namespace `{{ ctx.event_namespace }}`
 
 {% for event in ctx.events %}
-## Event `{{ event.name }}`
+## Event `{{ event.id }}`
```

Comment:
Just a question: Why using `id` and not `name` here?
Not blocking.

Reply:
Actually, the top level "name" will still exist on the `event` type, its just the `AnyValues` that will now only have the `id` (and no `name`)